[PLT-2583] Fix  image reference during cluster creation. Fix images r… (#819)
Stratio/kind: stratio-docs/en/modules/operations-manual/pages/offline-installation/common-images.adoc

= Common images

> Associated with the charts that _Stratio Cloud Provisioner_ installs in all clusters, the required images are:

Reviewer: ```suggestion
These are the required images used in the charts installed by _Stratio Cloud Provisioner_ across all clusters:
```
